tests/regression/runner.py

```diff
@@ -17,8 +18,8 @@
 # Utils:
 TIMEOUT = 100  # [sec]
 MSG_STATUS_CODE_405 = "[405] Method Not Allowed"
-CURL_EMPTY_REPLY = "000"
-
+MSG_STATUS_CODE_400 = "{\"message\":\"Invalid request header\"}"
```

Comment:
Why is this the only one in JSON format?

Reply:
The `MSG_STATUS_CODE_405` is the response of python module `requests`, and the other two error message are the response of curl.
I will remove `requests` and use curl for consistency.

`EMPTY_REPLY` is the the return status code of curl when no HTTP code was received due to a network error. And this is the issued you mentioned below.